doofinder-for-woocommerce/includes/api/class-store-api.php

```diff
@@ -373,4 +369,23 @@ private function build_callback_url($base_url, $endpoint_path)
         return $callback_url;
     }
 
+    /**
+     * This method throw_exception
+     *
+     * @param [type] $response
+     * @return void
+     */
+    private function throw_exception($response, $response_code)
+    {
+        if (is_wp_error($response)) {
+            $error_message = $response->get_error_message();
+            throw new Exception($error_message, (int)$response->get_error_code());
+        }
+
+        if ($response_code < 200 || $response_code >= 400) {
```

Comment:
Just a comment, instead of using these "magic numbers" directly (although I must confess that they are very common and well known), I suggest you to use this instead: `WP_Http.OK` or `WP_Http.BAD_REQUEST`. More info in https://developer.wordpress.org/reference/classes/wp_http/